resources/views/components/molecules/_m-listing----article.blade.php

```diff
@@ -6,6 +6,12 @@
                     @component('components.atoms._img')
                         @slot('image', $image ?? $item->imageFront('hero'))
                         @slot('settings', $imageSettings ?? '')
+                        @slot('class', 'img-hero-desktop')
+                    @endcomponent
+                    @component('components.atoms._img')
+                        @slot('image', $imageMobile ?? $item->imageFront('mobile_hero'))
+                        @slot('settings', $imageSettings ?? '')
+                        @slot('class', 'img-hero-mobile')
```

Comment:
For all these added mobile image compoenents we should fall back to the desktop images if there wasn’t a mobile one uploaded. So here it would be:

```
$imageMobile ?? $item->imageFront('mobile_hero') ?? $item->imageFront('hero')
```

In _m-listing----highlight.blade.php it would read something like:

```
$item->imageFront('mobile_hero') ?? $item->imageFront('hero') ?? $item->images[0]
```